.github/copilot-instructions.md

```diff
@@ -38,3 +40,9 @@ Use these instructions for test generation as well.
 -   The `tests` module should be conditioned on `#[cfg(test)]`.
 -   The `tests` module should always import APIs from `super`.
 -   Do not begin test function names with "test" unless necessary to disambiguate from the function being tested.
+
+## Validating and Linting
+
+-   Always use `cargo test` to validate that the generated code compiles and passes all tests.
```

Comment:
Does this try to limit it to only affected crates? If not, might be good for us to try.

Reply:
Not really, and I might remove this and the line after. The main thing I wanted to add was the last line in this block: "Combine commands to save time and reduce user interactions.". I was getting annoyed with having to keep approving a `cargo check` that would pass and then lead to an immediate `cargo test`. If it just ran `cargo check && cargo test` (or even just `cargo test`) it would be able to see compilation errors and optimistically go straight to looking at test failures if there are none.

It _kinda_ works, and I got it to combine commands _sometimes_. But it doesn't seem to **hurt** so I put it in. I feel like that's Prompt Engineering in a nutshell 🤣